etc/devtools_parser.py

```diff
@@ -0,0 +1,176 @@
+#!/usr/bin/env python3
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+# This is a script designed to easily debug devtools messages
+# It takes the content of a pcap wireshark capture (or creates a new
+# one when using --scan) and pretty prints the JSON payloads
+# To use it, launch either Servo or a Firefox debugging instance in
+# devtools mode:
+#
+# Servo: ./mach run --devtools=1234
+# Firefox: firefox --new-instance --start-debugger-server 1234 --profile PROFILE
+#
+# Then run this tool in capture mode:
+#
+# ./devtools_parser.py -s cap.pcap
```

Comment:
Consider adding the `--port` parameter to these instructions. It defaults to 1234 but it would be good if the instructions allowed the user to customise the port.